pkg/apis/monitoring/v1/types_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"testing"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/utils/ptr"
```

Comment:
let's keep the list of direct dependencies to the absolute minimum for this module.

Reply:
Got it. Thanks.